src/components/business/prevorder.jsx

```diff
@@ -1,9 +1,41 @@
 import React from "react";
+import PrevOrderItem from "./prevorderitem.jsx";
+import axios from "axios";
 
-class PreviousOrder extends React.Component {
+class PreviousOrders extends React.Component {
+  state = {
+    previousOrders: []
+  };
+
+  componentDidMount() {
+    this.getPreviousOrders();
+  }
+
+  getPreviousOrders() {
+    axios.get("http://localhost:7337/api/orders/complete").then(response => {
+      this.setState({
+        previousOrders: response.data
+      });
+      console.log('These are the previous orders: ', this.state.previousOrders);
+    });
+  }
+  
   render() {
-    return <h3>Previous Orders</h3>;
+    return (
+      <div className="previous-orders">
+        <h2>Previous Orders</h2>
+        {this.state.previousOrders.map((order, index) => {
+            return (
+              <div className="previous-order-item" key={index}>
```

Comment:
Passing in index as the key should be ok for our application but just bringing it up since it's an antipattern. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318